packages/host/tests/integration/components/ai-assistant-panel/codeblocks-test.gts

```diff
@@ -279,4 +279,115 @@ module('Integration | ai-assistant-panel | codeblocks', function (hooks) {
 
     await percySnapshot(assert);
   });
+
+  test('handles nested pre tags', async function (assert) {
+    let roomId = await renderAiAssistantPanel(`${testRealmURL}Person/fadhlan`);
+    let messageWithNestedPreTags = `I'll help you create an example with pre tags in HTML. Here's a useful example that demonstrates different ways to use the pre tag:
+\`\`\`html
+<!-- Basic pre tag example -->
+<pre>
+This is preformatted text
+    It preserves both spaces
+        and line breaks
+exactly as written
+</pre>
+
+<!-- Pre tag with code -->
+<pre>
+<code>
+function sayHello() {
+    console.log("Hello World!");
+}
+</code>
+</pre>
+
+<!-- Pre tag with styling -->
+<pre style="background-color: #f4f4f4; padding: 15px; border-radius: 5px;">
+const data = {
+    name: "John",
+    age: 30,
+    city: "New York"
+};
+</pre>
+
+<!-- Pre tag with HTML entities -->
+<pre>
+&lt;html&gt;
```

Comment:
Is this supposed to be `<html>...` and not `&lt;html&gt;...`? And similarly for the rest of the tags

Reply:
I copied this example from ai bot response and it seems everything under pre tag should be treated as text that is why the ai bot return `&lt;html&gt;` instead of `<html>`.